doc/releases/migration-guide-3.5.rst

```diff
@@ -235,6 +235,16 @@ Required changes
   This means that an extra linker stage is no longer necessary if this option is
   not enabled.
 
+* ZBus runtime observers implementation now relies on the HEAP memory instead of a memory slab. The
+  configuration changed. So, to keep your runtime observers code working correctly, you need to:
```

Comment:
The sentence
```
The configuration changed.
```
looks quite lonely. Is there something missing from it or perhaps more explanation could be added to it?

Reply:
For sure. I will adjust that.

Reply:
Done!